tox.ini

```diff
@@ -43,3 +43,30 @@ commands =
   stestr run {posargs}
   coverage3 combine
   coverage3 report
+
+[testenv:docs]
+envdir = .tox/docs
+basepython = python3
+commands =
+  sphinx-build -j auto -W -T --keep-going -b html {posargs} docs/ docs/_build/html
+
+[testenv:docs-clean]
+skip_install = true
+deps =
+allowlist_externals = rm
+commands = rm -rf {toxinidir}/docs/stubs/ {toxinidir}/docs/_build
+
+[testenv:gettext]
+envdir = .tox/docs
```

Comment:
Are you using translations for this new repo? If so, I recommend that being a dedicated follow up PR. So you should delete this for now. 

When deleting it, also delete line 48 of setting `envdir` for `docs`.

Reply:
I have no idea around translations - same for the apps now they are in community. Is the support wanted to be continued for that or not, if it is maybe adding it here seems reasonable too.

Reply:
> if it is maybe adding it here seems reasonable too.

This PR already has a lot of moving parts. So to help us increase the odds of getting everything right and giving good code review, I'd personally advocate for a follow up PR to deal with translations. Translations aren't essential for the general docs infra

Reply:
I was making a more general comment about translations as I am unsure about where things will stand with that - I am fine to remove this presently - Elena had been adding just what was needed. But since it will never be used it was to me a no-op at present so I was less than bothered by it being here - but removing it is fine - I was more concerned to getting the build failures sorted than finer details like this